### DISCUSSION TITLE
how to filter groups by assignment rule?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/psfalcon#230] [Q&A]
I tried this 
`Get-FalconHostGroup -Filter Get-FalconHostGroup -Filter "group_type: 'dynamic'"
`

tnx
nate

**Comment by bk-cs** [comment]
You can find dynamic groups using the `group_type` property:
```
Get-FalconHostGroup -Filter "group_type:'dynamic'"
```

There is no filter available to search by the `assignment_rule`. To work around that, you can retrieve all dynamic groups and use [Where-Object](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/where-object?view=powershell-7.2) to locally filter the results:
```
Get-FalconHostGroup -Filter "group_type:'dynamic'" -Detailed | Where-Object { $_.assignment_rule -match 'os_version' }
```
